### PR TITLE
First draft for "Contributions to PostgreSQL 11" blog post

### PR DESCRIPTION
**Pull request by andreasscherbaum**

### CODE REVIEW

File: content/post/Pivotal_Contributions_to_PostgreSQL_11.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+## Our contributors
+
+Although we have a number of people on all kind of PostgreSQL related projects, the following people contributed to the development of PostgreSQL 11:
+
+* Ashwin Agrawal (Catalog and Storage)
+* Asim Rama Praveen (Catalog and Storage)
+* Daniel Gustafsson
+* Heikki Linnakangas
+* Jacob Champion
+* Xin Zhang (Catalog and Storage)
```
Comment (danielgustafsson) [inline content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: Melanie Plageman has reviewed patches and should go here.

File: content/post/Pivotal_Contributions_to_PostgreSQL_11.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
++++
+authors = ["dgustafsson", "hlinnakangas"]
```
Comment (danielgustafsson) [inline content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: Should either by everyone listed in this blogpost, or just you. I vote for just your name.

Comment (andreasscherbaum) [reply content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: As mentioned on Slack already: I vote for everyone. Just missed this opportunity when I proofread the text. This blog post is about you guys, I'm just shadow writing it ;-)

Will follow up on that, and create profiles for everybody, if missing.

File: content/post/Pivotal_Contributions_to_PostgreSQL_11.md (hunk 1 above)
Comment (andreasscherbaum) [reply content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: Added Melanie to the list, removed the team names.

File: content/post/Pivotal_Contributions_to_PostgreSQL_11.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+
+
+## Outlook
+
+We have some ideas for upcoming PostgreSQL versions.
+
+Greenplum Database has table partitioning for a long time, that is a feature which was only recently added to PostgreSQL. We want to share some of the experiences we made around this feature, and enhance the functionality in PostgreSQL.
+
+Another existing limitation which we would like to remove is the length of text types. Currently that type can roughly hold 1 GB, but for some use cases that is not enough.
```
Comment (Eulerizeit) [inline content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: We aren't removing the limitation we are loosening the limitation.

Comment (andreasscherbaum) [reply content/post/Pivotal_Contributions_to_PostgreSQL_11.md]: Did change that, you are right.

**Pull request closed without merging**